### ISSUE TITLE
Sidebar's transient scrollbar doesn't fit in the Dark Theme

### ISSUE THREAD

**Issue by infinite-persistence** [lbryio/lbry-desktop#4727]
Having the default bright scrollbar for the Sidebar in the Dark Theme feels a bit jarring.  YT's dark theme uses a dark scrollbar.

Aside: It would be nice if the main scrollbar can be dark as well, but it's less distracting compared to the Sidebar's since the main scrollbar is on the far right.

**Comment by tzarebczan** [comment]
We were definitely aware of this one and how ugly it looks. Would you be able to take a stab? It uses the OS's theming so it may look fine on some. But we should have a standard experience similar to YT.

**Comment by infinite-persistence** [comment]
I found out that there was a previous attempt (PR 574) which was later removed.  My changes are pretty much similar to that.

Before proceeding further, were there technical reasons for the removal (and that I shouldn't repeat history)?  My guesses:
1. Too many browser-compatibility problems -- too much hassle, so might as well fallback to OS theme?
2. Removed due to css refactoring of variables, and simply didn't have time to refactor the scrollbar part, so temporarily remove the customization?
